src/features/usage-consolidation-accordion/usage-consolidation-accordion.js

```diff
@@ -84,12 +92,18 @@ const UsageConsolidationAccordion = ({
           onToggle={onToggle}
           headerProps={headerProps}
           contentRef={(n) => setAccordionContentRef(n)}
-        />
+        >
+          <UsageConsolidationFilters
+            onSubmit={(filterData) => onFilterSubmit(filterData)}
```

Comment:
```suggestion
            onSubmit={onFilterSubmit}
```

Reply:
I did it like this because a second parameter of submit handlers is a form object with some helpers like `initialize`, `submit`, `getState` etc. and I didn't want a consumer component to have access to them